.docs/README.md

```diff
@@ -253,17 +235,16 @@ Platební bránu je možné vytvořit pomocí formuláře, který najdete v [dok
 </form>
 ```
 
-#### Chyby s platbou
+#### Payment exception
+
+Two things can go wrong when calling `pay ()` or `payInline ()`:
 
-V okamžiku zavolání `pay()` nebo `payInline()` se mohou pokazit dvě věci:
+1. Bad parameters are provided somewhere
+2. The official Gopay payment gateway is broken
 
-1. Někde jsou poskytnuty špatné parametry
-2. Je pokažená oficiální platební brána Gopay
+The first mistake should never occur. It means a mistake in your code.
 
-První chyba by nikdy neměla nastat. Znamená totiž nějakou krpu ve vašem kódu.
-Druhá se může přihodit kdykoliv, proto generuje mírně odlišnou výjimku, kterou
-je třeba zachytit a podle ní informovat zákazníka, že chyba právě není na vaší
-straně.
+The second can happen at any time, so it generates a slightly different exception You need to catch and inform the customer according to it that the error is not up to you side.
```

Comment:
The second can happen at any time, so it generates a slightly different exception that You need to catch and inform the customer according to it that the error is not on to Your side.